src/openrct2/platform/Platform.Win32.cpp

```diff
@@ -590,67 +590,67 @@ namespace Platform
 
     uint16_t GetLocaleLanguage()
     {
-        CHAR langCode[4];
-
-        if (GetLocaleInfo(LOCALE_USER_DEFAULT, LOCALE_SABBREVLANGNAME, reinterpret_cast<LPSTR>(&langCode), sizeof(langCode))
-            == 0)
+        wchar_t langCode[LOCALE_NAME_MAX_LENGTH];
+        if (GetLocaleInfoEx(LOCALE_NAME_USER_DEFAULT, LOCALE_SNAME, langCode, static_cast<int>(std::size(langCode))) == 0)
         {
             return LANGUAGE_UNDEFINED;
         }
 
-        if (strcmp(langCode, "ENG") == 0)
+        auto localeStartsWith = [&langCode](const wchar_t* code) { return wcsncmp(langCode, code, std::wcslen(code)) == 0; };
+
+        if (localeStartsWith(L"en-GB"))
         {
             return LANGUAGE_ENGLISH_UK;
         }
-        if (strcmp(langCode, "ENU") == 0)
+        if (localeStartsWith(L"en-US"))
         {
             return LANGUAGE_ENGLISH_US;
         }
-        if (strcmp(langCode, "DEU") == 0)
+        if (localeStartsWith(L"de"))
         {
             return LANGUAGE_GERMAN;
         }
-        if (strcmp(langCode, "NLD") == 0)
+        if (localeStartsWith(L"nl"))
         {
             return LANGUAGE_DUTCH;
         }
-        if (strcmp(langCode, "FRA") == 0)
+        if (localeStartsWith(L"fr"))
         {
             return LANGUAGE_FRENCH;
         }
-        if (strcmp(langCode, "HUN") == 0)
+        if (localeStartsWith(L"hu"))
         {
             return LANGUAGE_HUNGARIAN;
         }
-        if (strcmp(langCode, "PLK") == 0)
+        if (localeStartsWith(L"pl"))
         {
             return LANGUAGE_POLISH;
         }
-        if (strcmp(langCode, "ESP") == 0)
+        if (localeStartsWith(L"es"))
         {
             return LANGUAGE_SPANISH;
         }
-        if (strcmp(langCode, "SVE") == 0)
+        if (localeStartsWith(L"sv"))
         {
             return LANGUAGE_SWEDISH;
         }
-        if (strcmp(langCode, "ITA") == 0)
+        if (localeStartsWith(L"it"))
         {
             return LANGUAGE_ITALIAN;
         }
-        if (strcmp(langCode, "POR") == 0)
+        if (localeStartsWith(L"pt-BR"))
```

Comment:
```suggestion
        if (localeStartsWith(L"pt"))
```

I’m pretty sure European Portuguese speakers would prefer Brazilian Portuguese to English.

Reply:
I don't think that's true, those languages are way more different than US and UK English for example, with different accents and meanings of words. While European Portuguese will most likely understand most of it, I don't think it's a good idea to throw Brazilian Portuguese at them on startup.

Reply:
As a native speaker of Brazilian Portuguese who has previously translated into European Portuguese thinking it would be fine, I can definitely tell you people will be very upset hahaha. So I don't think we should mix and match here.

Reply:
Very well then.